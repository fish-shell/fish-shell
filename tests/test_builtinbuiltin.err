builtin: Invalid combination of options,
--query and --names are mutually exclusive
