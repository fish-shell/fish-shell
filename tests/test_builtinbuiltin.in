# Tests for the "builtin" builtin/keyword.
builtin -q string; and echo String exists
builtin -q; and echo None exists
builtin -q string echo banana; and echo Some of these exist
builtin -nq string
echo $status
exit 0
