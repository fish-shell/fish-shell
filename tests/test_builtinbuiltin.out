String exists
Some of these exist
2
